xbmc/windowing/osx/WinEventsOSXImpl.mm

```diff
@@ -202,102 +202,40 @@ - (NSEvent*)InputEventHandler:(NSEvent*)nsEvent
     // handle mouse events and transform them into the xbmc event world
     case NSEventTypeLeftMouseUp:
     {
-      auto location = [self TranslateMouseLocation:nsEvent];
-      if (location.has_value())
-      {
-        NSPoint locationCoordinates = location.value();
-        newEvent.type = XBMC_MOUSEBUTTONUP;
-        newEvent.button.button = XBMC_BUTTON_LEFT;
-        newEvent.button.x = locationCoordinates.x;
-        newEvent.button.y = locationCoordinates.y;
-        [self MessagePush:&newEvent];
-      }
+      [self SendXBMCMouseEvent:nsEvent:newEvent:XBMC_MOUSEBUTTONUP:XBMC_BUTTON_LEFT];
```

Comment:
please add spaces around params as currently it's unreadable:
```objc
[self SendXBMCMouseEvent:nsEvent :newEvent :XBMC_MOUSEBUTTONUP :XBMC_BUTTON_LEFT];
```
each param on a single line is also probably a good idea since there're many